gateway/ln-gateway/src/lightning/cln.rs

```diff
@@ -139,4 +140,15 @@ impl ILnRpcClient for NetworkLnRpcClient {
         })?;
         Ok(res.into_inner())
     }
+
+    async fn update_scids(&self, scids: Vec<u64>) -> Result<EmptyResponse, LightningRpcError> {
+        let mut client = Self::connect(self.connection_url.clone()).await?;
+        let res = client
+            .update_scids(UpdateScidsRequest { scids })
+            .await
+            .map_err(|status| LightningRpcError::FailedToUpdateScids {
+                failure_reason: status.message().to_string(),
```

Comment:
Perhaps this gives more info?
```suggestion
                failure_reason: format!("{status:?}"),
```